Calculator/Calculator.cpp

```diff
@@ -20,35 +20,22 @@ bool char_is_num(const char c) {        //Return true if character c is a numera
     }
 }
 
-vector<string> str_to_strvec(string &equ_str) {
+vector<string> str_to_strvec(string& equ_str) {

```

Comment:
reference prefixes the variable name; sure it's of type "string reference", but, classic example: `std::string& s1, s2, s3` declares one string reference and 2 string variables where it appears to be declaring 3 references. to disambiguate this, the convention is to treat it as a prefix for the variable name rather than a suffix for the type

Reply:
so leave all references as `std::string &str`